php/php.editor/src/org/netbeans/modules/php/editor/completion/PHPCodeCompletion.java

```diff
@@ -335,7 +335,9 @@ public CodeCompletionResult complete(CodeCompletionContext completionContext) {
         // GH-4494 if query type is documentation, get a whole identifier as a prefix
         boolean upToOffset = queryType != QueryType.DOCUMENTATION;
         prefix = getPrefix(info, caretOffset, upToOffset, PrefixBreaker.WITH_NS_PARTS);
+        String prefixUntilCaret = getPrefix(info, caretOffset, true, PrefixBreaker.WITH_NS_PARTS); // GH-5881
```

Comment:
If `upToOffset` is `true`, we can skip this call and use `prefix` directly, no?

Reply:
Yes, You are right! Will fix it. Thanks.
```java
String prefixUntilCaret = upToOffset ? prefix : getPrefix(info, caretOffset, true, PrefixBreaker.WITH_NS_PARTS); // GH-5881
```

Reply:
Fixed: https://github.com/apache/netbeans/compare/1fcbaa35a3af943c7db9a4ad40fe9d4531f56a69..946bd31030a69df973f41e94f458af61c38a9ac5